pkg/cli/interactive_tests/test_missing_log_output.tcl

```diff
@@ -63,6 +63,10 @@ eexpect ":/# "
 
 # Start a regular server
 send "$argv start >/dev/null 2>&1 & sleep 1\r"
+eexpect "\[1\]"
+eexpect ":/# "
+# Give the server some time to start.
+sleep 0.5
```

Comment:
we do -- `start --background` waits until the server is ready, using systemd's readiness signaling.

concretely, sleeping a fixed duration this seems likely to flake when CIs / docker scheduler is unusually slow and we happen to cross the 0.5s, or 1s, etc mark, but if we make it so long we never hit it, we have a slow test.